models/aged_creditor.json

```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "aged_creditor_facts",
     "label": "Aged Creditor Analysis",
-    "description": "This is based on the AC and ACA return forms.",
+    "description": "Analysis of creditors by age of debt.",
```

Comment:
Worth adding "This does not list individual creditors, but rather the amount of debt in each age group for each line item category"?